org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/extensions/contentmodel/participants/codeactions/src_import_1_2CodeAction.java

```diff
@@ -0,0 +1,89 @@
+/*******************************************************************************
+* Copyright (c) 2019 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+
+package org.eclipse.lsp4xml.extensions.contentmodel.participants.codeactions;
+
+import java.util.List;
+
+import org.eclipse.lsp4j.CodeAction;
+import org.eclipse.lsp4j.Diagnostic;
+import org.eclipse.lsp4j.Range;
+import org.eclipse.lsp4xml.commons.BadLocationException;
+import org.eclipse.lsp4xml.commons.CodeActionFactory;
+import org.eclipse.lsp4xml.dom.DOMDocument;
+import org.eclipse.lsp4xml.dom.DOMElement;
+import org.eclipse.lsp4xml.dom.DOMNode;
+import org.eclipse.lsp4xml.services.extensions.ICodeActionParticipant;
+import org.eclipse.lsp4xml.services.extensions.IComponentProvider;
+import org.eclipse.lsp4xml.settings.XMLFormattingOptions;
+import org.eclipse.lsp4xml.utils.XMLPositionUtility;
+
+/**
+ * Code action to fix cvc-attribute-3 error.
+ *
+ */
+public class src_import_1_2CodeAction implements ICodeActionParticipant {
+
+	@Override
+	public void doCodeAction(Diagnostic diagnostic, Range range, DOMDocument document, List<CodeAction> codeActions,
+			XMLFormattingOptions formattingSettings, IComponentProvider componentProvider) {
+
+		try {
+			String prefix = document.getDocumentElement().getPrefix();
+			CodeAction namespace = createNamespaceCodeAction(diagnostic, range, document, prefix);
+			CodeAction targetNamespace = createTargetNamespaceCodeAction(diagnostic, document, prefix);
+
+			if (namespace != null) {
+				codeActions.add(namespace);
+			}
+
+			if (targetNamespace != null) {
+				codeActions.add(targetNamespace);
+			}
+
+		} catch (BadLocationException e) {
+			// Do nothing
+		}
+	}
+
+	private CodeAction createNamespaceCodeAction(Diagnostic diagnostic, Range range, DOMDocument document, String prefix) throws BadLocationException {
+		int offset = document.offsetAt(range.getStart());
+		DOMNode node = document.findNodeAt(offset);
+
+		if (node == null || !node.isElement()) {
+			return null;
+		}
+		
+		String message;
+		if (prefix != null) {
+			message = "Insert namespace attribute in <" + prefix + ":import> element";
```

Comment:
I don't know why but <xs:import> doesn't appear. In my case:

![image](https://user-images.githubusercontent.com/1932211/61331859-1324a080-a823-11e9-8af9-f5be08b0f640.png)

To avoid having this problem, I suggest you to change _<_ with `'`, and namespace should be enclosed with  `'`. In other words, I suggest to change with:

`"Insert 'namespace' attribute in '" + prefix + ":import' element"`